code/modules/events/portal_storm.dm

```diff
@@ -1,133 +0,0 @@
-/datum/round_event_control/portal_storm_syndicate
-	name = "Portal Storm: Syndicate Shocktroops"
-	typepath = /datum/round_event/portal_storm/syndicate_shocktroop
-	weight = 2
-	earliest_start = 18000
-
-/datum/round_event/portal_storm/syndicate_shocktroop
-	boss_types = list(/mob/living/simple_animal/hostile/syndicate/melee/space/stormtrooper = 2)
-	hostile_types = list(/mob/living/simple_animal/hostile/syndicate/melee/space/noloot = 8,\
-						/mob/living/simple_animal/hostile/syndicate/ranged/space/noloot = 2)
-
-/datum/round_event_control/portal_storm_narsie
-	name = "Portal Storm: Nar-sie"
-	typepath = /datum/round_event/portal_storm/portal_storm_narsie
-	weight = -1
```

Comment:
The event subsystem is undocumented garbage, and this is the kinda error that supposed to be caught by a maintainer.